### ISSUE TITLE
Textarea's are not full width under k-form class

### ISSUE THREAD

**Issue by davidboothe** [telerik/kendo-angular#721]
form fields should be stretched to 100% when you use the k-form class on the form. Input fields are 100% however textarea's are not.

textarea's are set to a width of 18em.

**Comment by gyoshev** [comment]
Indeed, currently the widths of inputs and the textarea are set to values that appear to be arbitrary. This will require some rigorous testing to ensure that using 100% works well across the theme, but it will remove some of the magic in the theme and is worth it.
